shared/constants/selectors.js

```diff
@@ -30,6 +30,16 @@ const userIsInTeam = (
   return teamNameToMemberUsernames.getIn([teamname, username])
 }
 
+const userIsActiveInTeam = (
+  {entities: {teams: {teamNameToMembers}}}: TypedState,
+  teamname: string,
+  username: string
+) => {
+  const members = teamNameToMembers.get(teamname, Set())
+  const member = members.find(mem => mem.username === username)
```

Comment:
not ideal that this is an object which basically makes it being in the set and us always doing finds O(N)

Reply:
we don't have to change this here, but noting this is a code smell